Exiled.Events/Patches/Events/Player/SendingAdminChatMessage.cs

```diff
@@ -54,7 +54,10 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
                     // message
                     new(OpCodes.Ldarg_0),
 
-                    // SearchingPickupEventArgs ev = new(Player, ItemPickupBase, SearchSession, SearchCompletor, float);
+                    // true
+                    new(OpCodes.Ldc_I4_1),
+
+                    // SearchingPickupEventArgs ev = new(Player, string, bool);
```

Comment:
oh stop why it is SearchingPickupEventArgs

Reply:
because yamato does copy past really dumbly